### DISCUSSION TITLE
Torrent or P2P Traffic Detected

### DISCUSSION THREAD

**Discussion by ccmks** [SmoothWAN/SmoothWAN#120] [General & Help]
Hello,

I am not sure if you can help on this. My speedify keep disconnecting because it says it is detected torrent or p2p traffic. However, I am not using those at all. How speedify identify my traffic as torrent or p2p traffic so I can filter those packet out and not to use speedify network?

Thank you

**Comment by TalalMash** [comment]
In the latest release, I added this https://smoothwan.com/tips/#identifying-client-using-p2p-or-bittorrent to identify the clients using BitTorrent.
![image](https://user-images.githubusercontent.com/96490382/219457316-83580a37-7ddc-49bb-90df-9289f3a08b4f.png)

Speedify is tame with Windows P2P and other non torrenting traffic.

I also added "ntopng" which is a heavy tool, but is able to identify the type of traffic in very high details using DPI, you can find and install it in `Services` menu (around 400-500 MB in size)

**Comment by ccmks** [comment]
I was using that tool earlier but didn't show anything on BitTorrent activity log, but speedify still flag my connection as P2P

**Comment by TalalMash** [comment]
It's probably not torrent, I'm not aware of what Speedify considers P2P, I wish there was a list in support page.
You can try using `ntopng` in services menu, and select `br-lan`, then `Live flows` in the sidebar.

**Comment by ccmks** [comment]
I contact speedify support and they want me to generate log. How can I generate the log?

I tried to click generate log in UI but nothing happens

**Comment by TalalMash** [comment]
In SmoothWAN Speedify menu (where you install) there is a generate log button.

**Comment by TalalMash** [comment]
Speedify related, move to discussions.
